packages/react/src/themes/teams/withProcessedIcons.ts

```diff
@@ -12,21 +12,33 @@ const declareSvg = (svgIcon: SvgIconSpec, exportedAs?: string): ThemeProcessedIc
   exportedAs,
 })
 
-const processedIcons: ThemeIcons = Object.keys(svgIconsAndStyles as {
+const getIcon = iconAndMaybeStyles => {
```

Comment:
given that it was decided to extract this method, it would be beneficial if we will specify types for the args and return value (actually, the return value type may be the most beneficial piece)